src/Microsoft.Azure.WebJobs.Host/Config/ExtensionConfigContext.cs

```diff
@@ -16,8 +16,8 @@ public class ExtensionConfigContext : FluentConverterRules<Attribute, ExtensionC
         // List of actions to flush from the fluent configuration. 
         private List<Action> _updates = new List<Action>();
 
-        // track which TAttribute rules have been added. 
-        private HashSet<object> _existingRules = new HashSet<object>();
+        // Map of tyepof(TAttribute) --> FluentBindingRule<TAttribute>
```

Comment:
Can the dictionary be strongly typed then?

Reply:
No. Because the objects have a generic signature. 

---
In reply to: [151523014](https://github.com/Azure/azure-webjobs-sdk/pull/1422#discussion_r151523014) [](ancestors = 151523014)